### ISSUE TITLE
What to do this week...

### ISSUE THREAD

**Issue by nlottig94** [ebeshero/EmilyDickinson16#6]
Dr. B. told me that she wants us all to experiment with trying to make a table of contents list or table by using XSLT  to filter through all of the Emily Dickinson files that we downloaded today from GitHub.  She said to look over XSLT Exercise 5.  Also, she said someone could create a CSS stylesheet for it as well!  Is this all, @ebeshero ??

**Comment by nlottig94** [comment]
I created an XSLT to output a table on html that displays all of the variants of the entire collection of poems.  I had the Dickinson folder that holds all of the poems on my local desktop where I initially had my XSLT saved! @ebeshero @amielnicki @blawrence719 @brookestewart

**Comment by nlottig94** [comment]
I also pushed my output html to GitHub too!!

**Comment by brookestewart** [comment]
Great! I'll get started on creating a css template.

**Comment by blawrence719** [comment]
I am going to start work on a homepage.

**Comment by nlottig94** [comment]
I pushed the Dickinson folder that is holding the collection of xml poems to GitHub, so I also tweaked my table xslt a bit.  I then went ahead and put all we did last night onto our web space.  If someone wants to try to link everything together?  Also, I believe @amielnicki is going to look through all of the poems to make sure the transcriptions are correct.  I make try to tweak the table a little bit more for next week, but if someone else wants to give it a try, go for it!  Have a good weekend, everyone!!
